gapis/service/service.proto

```diff
@@ -420,6 +421,21 @@ message MemoryRange {
   uint64 size = 2;
 }
 
+// UsageHints hints to the server the intended usage of the result of a request.
+// This can be used to improve performance and responsiveness of the RPCs.
+message UsageHints {
```

Comment:
> should this be an enum instead?

Not sure these two are mutually exclusive. It is possible you have a grid of thumbnails and one that is the primary. Like the selected object.

> If we'd ever want to add something variable, wouldn't it be part of the RenderSettings?

No, I intend for `UsageHints` to be passed to other RPCs in the future, that may not have anything to do with rendering.